docs/guide/upgrading.md

```diff
@@ -56,25 +57,27 @@ Your project specific modules should not use the `@apostrophecms` namespace to a
-  - Many module options that were top level properties in A2 (e.g., `label`, `alias`, `sort`) are now subproperties of `options`. These include all options with primitive value (e.g., booleans, string, numbers) in addition to select object or array options. See the [module options reference](/reference/module-api/module-options.md).
+  - Many module options that were top level properties in A2 (e.g., `label`, `alias`, `sort`) are now subproperties of `options`. These include all settings that are *not* covered in the [module properties overview reference](/reference/module-api/module-overview.md). See the [module options reference](/reference/module-api/module-options.md) for more on that.
+  <!-- TODO: Update once options are all moved into individual module reference pages -->
   - The A2 `self.addHelper()` method used to add Nunjucks helper functions is replaced by the `helpers` section in module configuration.
   - The A2 `self.apiRoute()` method used to add a custom API route is replaced by the `apiRoutes` section in module configuration.
   - The A2 `self.renderRoute()` method used to add a custom template rendering route is replaced by the `renderRoutes` section in module configuration.
   - The A2 `self.route()` method used to add a custom Express route is replaced by the `routes` section in module configuration.
   - The A2 `self.on()` method that watched for Apostrophe server-side promise events is replaced by the `handlers` section in module configuration.
   - The A2 `self.addFilter()` method used to add MongoDB cursor filters is replaced by the `queries` section and its `builders` subsection in module configuration.
-  - The A2 `self.expressMiddleware()` method for adding Express middleware functions is replaced by the `middleware` section in module configuration.
+  - Custom `lib/cursor.js` files for modules are no longer loaded. Migrate this code to the new `queries` section as well.
+  - The A2 `self.expressMiddleware` system for adding Express middleware functions is replaced by the `middleware` section in module configuration.
   - The A2 `self.addTask()` method for adding CLI tasks is replaced by the `tasks` section in module configuration.
 
 ### Areas and pages
 
 - Every page type will need a corresponding module (in A2 this was only necessary if the page had custom fields or functionality). Page templates live in the page type module rather than in the base page module.
 - Areas must be declared as fields in the [content field schema](/guide/content-schema.md). They will no longer work if simply added to template files without being registered.
-- The area template tag format is now `{% area data.page, 'areaName' %}`. Note that this no longer includes the object of widget configuration since that is now done in module config.
+- The area template tag format is now `{% area data.page, 'areaName' %}`. Note that this no longer includes the object of widget configuration since that is now done in module configuration.
```

Comment:
How about "no longer includes the configuration of widgets," seems a little needlessly technical the other way.